.github/workflows/env/production

```diff
@@ -0,0 +1,17 @@
+# All referenced variables must be set or the script will fail.
+BASELIME_API_KEY=${BASELIME_API_KEY}
+DX_IPDATA_API_KEY=${IPDATA_API_KEY}
+IPFS_API_SECRET=${IPFS_API_SECRET}
+NPM_TOKEN=${NPM_TOKEN}
+SENTRY_RELEASE_AUTH_TOKEN=${SENTRY_RELEASE_AUTH_TOKEN}
+CLI_SENTRY_DESTINATION=${CLI_SENTRY_DESTINATION}
+CLI_SEGMENT_API_KEY=${CLI_SEGMENT_API_KEY}
+COMPOSER_APP_SENTRY_DESTINATION=${COMPOSER_APP_SENTRY_DESTINATION}
+COMPOSER_APP_SEGMENT_API_KEY=${COMPOSER_APP_SEGMENT_API_KEY}
+DX_DISCORD_WEBHOOK_URL=${DX_DISCORD_WEBHOOK_URL}
+CLOUDFLARE_API_TOKEN=${CLOUDFLARE_API_TOKEN}
+CLOUDFLARE_ACCOUNT_ID=${CLOUDFLARE_ACCOUNT_ID}
+AGENT_HOSTING_SERVER="https://api.prod.agenthosting.dxos.network:8082/v1alpha1/"
+AGENT_HOSTING_TYPE=AGENTHOSTING_API
+# End common vars
+JWT_SECRET=${JWT_SECRET}
```

Comment:
is this used to decode the cookie? this will go away once you update that agent code?